boards/cats_rev1Pro/src/cli/cli.c

```diff
@@ -259,13 +262,20 @@ static void cliDefaults(const char *cmdName, char *cmdline) {
 
 static void cliDump(const char *cmdName, char *cmdline) {}
 
-static void cliExit(const char *cmdName, char *cmdline) {}
+static void cliExit(const char *cmdName, char *cmdline) {
+  NVIC_SystemReset();
```

Comment:
when is cliExit called and what does NVIC_SystemReset do?

Reply:
It is called to reset the program, NVIC_SystemReset resets the processor :)